impl/src/test/java/org/glassfish/el/test/OperatorTest.java

```diff
@@ -39,22 +39,14 @@ public static void tearDownClass() throws Exception {
     public void setUp() {
     }
     
-    void testExpr(String testname, String expr, Long expected) {
+    void testExpr(String testname, String expr, Object expected) {
         System.out.println("=== Test " + testname + " ===");
```

Comment:
Did you have to change the Long to Object, to resolve a failure?

Reply:
No, it was done just to re-use the same method and not create a new one with Boolean. If you see the same method is now used for all the tests.